metadata_mapper/tests.py

```diff
@@ -1,19 +1,59 @@
-from lambda_function import map_page
-# from validate_mapping import validate_mapped_collection
-from sample_data.nuxeo_harvests import *
-from sample_data.oac_harvests import *
-from sample_data.islandora_harvests import *
 import json
+import argparse
+import settings
+import logging
+import os
+from lambda_shepherd import map_collection
+from validate_mapping import validate_mapped_collection
+from sample_data.nuxeo_harvests import nuxeo_harvests, \
+    nuxeo_complex_object_harvests, nuxeo_nested_complex_object_harvests
+from sample_data.oac_harvests import oac_harvests
+from sample_data.islandora_harvests import islandora_harvests
+from map_registry_collections import map_endpoint
+from validate_registry_collections import validate_endpoint
 
-harvests = islandora_harvests
 
-for harvest in harvests:
-    print(f"mapping tests: {json.dumps(harvest)}")
-    map_page(json.dumps(harvest), {})
-    print(f"mapped: {str(harvest)}")
+def main():
+    vernacular_path = settings.local_path(
+        'vernacular_metadata', 0)[:-1]
+    urls = [
+        f"https://registry.cdlib.org/api/v1/rikoltimapper/{f}/?format=json"
+        for f in os.listdir(vernacular_path)
```

Comment:
this now just maps against everything in the local filesystem's vernacular metadata path.

Reply:
oh yeah that's nicer